var/spack/repos/builtin/packages/wrf/package.py

```diff
@@ -237,6 +240,13 @@ def setup_build_environment(self, env):
             env.set("HDF5", self.spec["hdf5"].prefix)
             env.prepend_path("PATH", ancestor(self.compiler.cc))
 
+    def flag_handler(self, name, flags):
+        # Same flags as FCFLAGS/FFLAGS above, but forced through the compiler
+        # wrapper when compiling v3.9.1.1.
+        if self.spec.satisfies("@3.9.1.1 %gcc@10:") and name == "fflags":
+            flags.append("-w -O2 -fallow-argument-mismatch -fallow-invalid-boz")
```

Comment:
```suggestion
            flags.extend(["-w", "-O2", "-fallow-argument-mismatch", "-fallow-invalid-boz"])
```

not that it matters much since i believe we go to string anyways, but this would in the future allow filtering of flags better.